src/components/Button.tsx

```diff
@@ -11,11 +12,19 @@ type GradientDuoTone =
   | 'pinkToOrange'
   | 'tealToLime'
   | 'redToYellow';
-
+type Size = 'extraSmall' | 'small' | 'medium' | 'large' | 'extraLarge';
+type IconPosition = 'left' | 'right';
 export type ButtonProps = ComponentProps<'button'> & {
   pill?: boolean;
   outline?: boolean;
+  disabled?: boolean;
```

Comment:
I don't see why `disabled` should be mentioned since it is already included in the type `ComponentProps<'button'>`!